django/core/validators.py

```diff
@@ -397,8 +397,12 @@ class StepValueValidator(BaseValidator):
     message = _("Ensure this value is a multiple of step size %(limit_value)s.")
     code = "step_size"
 
+    def __init__(self, limit_value, offset=None, message=None):
```

Comment:
Would this be better? Imho I think at least if the default's in the signature then it's clear to users of the validator + it goes into documentation:

```suggestion
    def __init__(self, limit_value, offset=0, message=None):
```

Reply:
I'd leave `None`, then you can always pass `None` to get the default value, i.e. `0`.